core/src/main/scala/cats/Reducible.scala

```diff
@@ -112,6 +112,12 @@ import simulacrum.typeclass
       val F = self
       val G = Reducible[G]
     }
+
+  def minimum[A](fa: F[A])(implicit A: Order[A]): A =
```

Comment:
I did this specifically to avoid collision with the std lib. If I refactor something from a `NonEmptyList` to a `List` I'd prefer to get a compile error instead of something with no type signature change but that will start throwing exceptions at runtime.

Having said that, there's a question of how much we should bend over backwards to avoid collisions with std lib names. I'm not set in stone on this if people would prefer `min`.